pyopenetl/operations.py

```diff
@@ -304,7 +304,38 @@ def delete_table(self, table: str) -> None:
         with self.dest_conn.connect() as cloud_sql_conn:
             print(f"--> Deleting table {table}")
             cloud_sql_conn.execute(f"DROP TABLE IF EXISTS {table}")
+    def get_indices_from_heroku(self, read_table: str, write_table: str) -> str:
+        """
+        Reads the index commands from the source heroku table, and returns a command to run on the destination table.
+
+        args:
+            read_table: the name of the heroku table being read from
+            write_table: the name of the Cloud SQL table we write to
+        """
 
+        # Query to get indices from read_table
+        data_to_resolve_query = f"""SELECT
+                                tablename,
+                                indexname,
+                                indexdef
+                            FROM
+                                pg_indexes
+                            WHERE
+                                schemaname = 'public' AND
+                                tablename = '{read_table}'
+                            ORDER BY
+                                tablename,
+                                indexname;"""
+        with self.source_conn.connect() as conn:
+            index_df = pd.read_sql(data_to_resolve_query, conn)
```

Comment:
I think you'll see a warning from pandas here if you don't use kwargs for everything other than the query text 
```suggestion
            index_df = pd.read_sql(data_to_resolve_query, con=conn)
```